src/codeflare_sdk/ray/cluster/cluster.py

```diff
@@ -176,6 +185,45 @@ def up(self):
         except Exception as e:  # pragma: no cover
             return _kube_api_error_handling(e)
 
+    # Applies a new cluster with the provided or default spec
+    def apply(self, force=False):
+        """
+        Applies the Cluster yaml using server-side apply.
+        If 'force' is set to True, conflicts will be forced.
+        """
+        # check if RayCluster CustomResourceDefinition exists if not throw RuntimeError
+        self._throw_for_no_raycluster()
+        namespace = self.config.namespace
+        name = self.config.name
+        try:
+            self.config_check()
+            api_instance = client.CustomObjectsApi(get_api_client())
+            crds = self.get_dynamic_client().resources
+            if self.config.appwrapper:
+                api_version = "workload.codeflare.dev/v1beta2"
+                api_instance = crds.get(api_version, kind="AppWrapper")
```

Comment:
```suggestion
                api_instance = crds.get(api_version=api_version, kind="AppWrapper")
```
This should prevent a `TypeError` when trying to create an AppWrapper via `apply()`

Reply:
Even with the update to the api_instance there are still errors when trying to apply AppWrappers.
```
Response: b'{"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"PatchOptions.meta.k8s.io \\"\\" is invalid: fieldManager: Required value: is required for apply patch","reason":"Invalid","details":{"group":"meta.k8s.io","kind":"PatchOptions","causes":[{"reason":"FieldValueRequired","message":"Required value: is required for apply patch","field":"fieldManager"}]},"code":422}\n'
```

Reply:
I added the field-manager field to the server_side_apply call (for AppWrapper). That should have fixed it.
I cannot reproduce it now, and I see the AppWrapper being properly created.